_tools/format.js

```diff
@@ -1,6 +1,7 @@
-#!/usr/bin/env -S deno run --allow-read --allow-run
-const p = Deno.run({
-  cmd: [Deno.execPath(), "fmt", "--config", "deno.json", ...Deno.args],
-});
-const status = await p.status();
-Deno.exit(status.code);
+#!/usr/bin/env -S deno run --no-check
+import * as semver from "https://deno.land/x/semver@v1.0.0/mod.ts";
+if (semver.lt(Deno.version.deno, "1.18.0")) {
+  console.log("This script is deprecated. Upgrade to the latest Deno and run `deno fmt` command directly");
```

Comment:
Although, maybe we should actually keep the script running instead of doing nothing here?

Reply:
Ah, ok. Let's do it